src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveService.kt

```diff
@@ -112,6 +112,12 @@ class OppgaveService(
     }
 
     fun fordelOppgave(gsakOppgaveId: Long, saksbehandler: String): Long {
+
+        val gsakOppgave = hentOppgave(gsakOppgaveId)
+        val tidligereSaksbehandler = gsakOppgave.tilordnetRessurs
+        if (tidligereSaksbehandler != saksbehandler && tidligereSaksbehandler != null) {
+            logger.info("(Eier av behandling/oppgave) Fordeler OppgaveId: $gsakOppgaveId som hadde tidligere saksbehandler $tidligereSaksbehandler til $saksbehandler")
```

Comment:
Trenger vi den parentesen? 
```suggestion
            logger.info("Fordeler oppgave=$gsakOppgaveId fra tidligere saksbehandler $tidligereSaksbehandler til $saksbehandler")
```

Reply:
Oki.
Hadde kanskje vært fint å få med tid siden oppgaven sist ble endret, sånn att man kan finne ut om en person går inn på en oppgave som nettopp blitt tildelt? (kan være interessant for oppgaver som ikke har eier også?)